CHANGELOG.md

```diff
@@ -1,4 +1,4 @@
-# Changelog
+# VRO Changelog
```

Comment:
This is auto-generated isn't it?

Reply:
That was just a small update to trigger a new pipeline run so I could test a change in the .github repo workflow lint pipeline.